lib/ramble/docs/tutorials/modifying_a_gromacs_experiment.rst

```diff
@@ -0,0 +1,221 @@
+.. Copyright 2022-2023 Google LLC
+
+   Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+   https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+   <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+   option. This file may not be copied, modified, or distributed
+   except according to those terms.
+
+.. _modifying_an_experiment_tutorial:
+
+=======================================
+3) Modifying A GROMACS Experiment
+=======================================
+
+In this tutorial, you will learn how to modify an existing workspace
+configuration that contains experiments using
+`GROMACS <https://www.gromacs.org/>`_, a free and open-source application
+for molecular dynamics.
+
+This tutorial builds off of concepts introduced in previous tutorials. Please
+make sure you review those before starting with this tutorial's content.
```

Comment:
Do we need an explicit call out like "go setup the workspace from tutorial 2?"

Reply:
I think this should be there now (It's maybe hard to see when you look at the source, but there's an include line that pulls in the same "Create and configure the workspace" text as in tutorial 2.